os/tools/compression/src/compress.c

```diff
@@ -65,41 +66,47 @@ static void compress_file(int block_size, int type, char *in_file, char *out_fil
 #if CONFIG_COMPRESSION_TYPE == LZMA
 	long unsigned int propsSize = LZMA_PROPS_SIZE;
 #endif
+	bool compression_fail = false;
```

Comment:
@abhishek-samsung Let me suggest one thing for readability. Instead of compression_fail, what about using compression_success?
Our purpose and intention is to make compression successful, not to get failed.
So, it might be better for readability to use compression_success = true/false rather than compression_fail = true/false.
It's up to you.

Reply:
@abhishek-samsung Could you let us know your opinion about @davidfather 's comment?

Reply:
I will change compression_fail to compression_success as @davidfather suggested.